docs/using-ecs/guidelines/implementation.asciidoc

```diff
@@ -226,5 +226,5 @@ appeared:
 
 [source,sh]
 ----
-related.ip: 10.42.42.42
+related.ip: [ "10.42.42.42" ]
```

Comment:
```suggestion
related.ip: ["10.42.42.42"]
```

Those spaces are unusually looking to me, but there's nothing wrong with them from YAML P.O.V.